libs/client/EventHandler.lua

```diff
@@ -537,4 +537,18 @@ function EventHandler.WEBHOOKS_UPDATE(d, client) -- webhook object is not provid
 	return client:emit('webhooksUpdate', channel)
 end
 
+function EventHandler.INTERACTION_CREATE(d, client)
+	local url = "https://discord.com/api/v8/interactions/"..d.id.."/"..d.token.."/callback"
+	local headers = {{"Content-Type", "application/json"}}
+	local payload = "{\"type\": 6}"
+	local http = require("coro-http")
+	http.request("POST", url, headers, payload)
```

Comment:
Making a HTTP request behind the scenes the exact kind of scenario that discordia goes out of it's way to avoid. This shouldn't be here.